dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/security/AuthenticationControllerTest.java

```diff
@@ -57,6 +57,21 @@ class AuthenticationControllerTest extends AuthenticationApiTestBase {
   @Autowired SystemSettingManager systemSettingManager;
   @Autowired private SessionRegistry sessionRegistry;
 
+  @Test
+  void testSuccessfulLoginWithOldUsername() {
+    User adminUser = userService.getUserByUsername("admin");
+    adminUser.setUsername("Üsername");
```

Comment:
does the system allow you to set a username to an invalid username? is this a separate bug?

Reply:
This is only possible manually like I do, not trough the API.